tests/utils_test.py

```diff
@@ -511,6 +508,19 @@ def snippets_to_list(cls, snippets):
 
         return snippets_list
 
+    @classmethod
+    def relationships_to_list(cls, relationships: List[Relationship]) -> List:
```

Comment:
Maybe `relationships_to_dict_list`? I find it confusing as-is because the provided parameter is already a list... And the returned type can be `List[Dict]`, right? Maybe even `List[OrderedDict]` if the ordering is relevant, I can't really tell

Reply:
fixed